plextraktsync/config/HttpCacheConfig.py

```diff
@@ -30,6 +30,28 @@ class HttpCacheConfig:
         "*.trakt.tv/users/*/watchlist/shows": DO_NOT_CACHE,
         "*.trakt.tv/users/likes/lists": DO_NOT_CACHE,
         "*.trakt.tv/users/me": DO_NOT_CACHE,
+
+        # Plex patterns
+        # Ratings search
+        "*/library/sections/*/all?*userRating%3E%3E=-1*": DO_NOT_CACHE,
+        # fetch_item
+        "*/library/sections/*/all?*": DO_NOT_CACHE,
+        "*/library/sections/*/collections": DO_NOT_CACHE,
+        "*/library/metadata/*": DO_NOT_CACHE,
```

Comment:
I think `metadata.provider.plex.tv/library/metadata/*` should be cached because there is no user data in response.
But `metadata.provider.plex.tv/library/metadata/*/userState` must remain not cached.

Reply:
Added dc07f64f, but perhaps should have still some cache expiry rather `NEVER_EXPIRE`? how much?

Reply:
If we could have a random value, that'd be perfect.
It would allow to spread out the requests.
E.g. 3 month +random(0-30days)

Reply:
b667c0eb, 5f2243b1.

but how to test that? enable all watchlist related options and like some online movie?

Reply:
there's no `metadata.provider.plex.tv/library/metadata/*/userState`, but rather `https://metadata.provider.plex.tv/library/metadata/*?*includeUserState=1` and it includes view counts etc.

but I don't think we use those values, only test for guids

Reply:
randomize expire request:
- https://github.com/requests-cache/requests-cache/issues/768